supplier/tests/test_supplier.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from conftest import Fields, TestFields
+from supplier import models
+
+
+class TestSupplierModel:
+    def test_supplier_creation(self, supplier_test):
```

Comment:
This doesn't really test anything. we need to check for the fields in the DB, after inserting the fixture to it. not the fields of the fixture. we created it, we should know it works.